content/en/metrics/units.md

```diff
@@ -123,3 +123,4 @@ Time units between a minute and a year are split into multiple units to be more
 
 [1]: https://app.datadoghq.com/metric/summary
 [2]: https://en.wikipedia.org/wiki/Metric_prefix#List_of_SI_prefixes
+[3]: 
```

Comment:
Looks like you forgot the link target? :)

Reply:
Embarrassing ^_^

```suggestion
[3]: https://docs.datadoghq.com/dashboards/guide/unit-override/
```